packages/lib-classifier/src/store/SubjectViewerStore.spec.js

```diff
@@ -1,12 +1,37 @@
 import asyncStates from '@zooniverse/async-states'
+import { Factory } from 'rosie'
+import SubjectStore from './SubjectStore'
 import SubjectViewerStore from './SubjectViewerStore'
 
+const subject = Factory.build('subject')
+
 describe('Model > SubjectViewerStore', function () {
   it('should exist', function () {
     expect(SubjectViewerStore).to.be.ok()
     expect(SubjectViewerStore).to.be.an('object')
   })
 
+  describe('frame', function () {
+    let subjectViewerStore
+
+    beforeEach(function () {
+      subjectViewerStore = SubjectViewerStore.create()
+    })
+
+    it('should have a `frame` property', function () {
+      expect(SubjectViewerStore.frame).to.equal(undefined)
```

Comment:
I'd expect this to equal `0` with `types.optional(types.integer, 0)`, but it's undefined.